js/main.js

```diff
@@ -26,6 +28,18 @@ function storeValues(event) {
   placeHolder.setAttribute('src', './images/placeholder-image-square.jpg');
   viewSwap('entries');
   toggleNoEntries();
+
+  if (data.editing !== null) {
+    for (var i = 0; i < data.entries.length; i++) {
+      if (data.editing.entryId === data.entries[i].entryId) {
+        data.entries[0].entryId = data.entries[i].entryId;
+        data.entries.splice([i], 1);
+      }
+
```

Comment:
When you edit an entry, is its DOM tree getting updated with the new information without needing to refresh? If not, it needs to.